src/pytom_tm/matching.py

```diff
@@ -189,16 +208,14 @@ def run(self) -> tuple[npt.NDArray[float], npt.NDArray[float], dict]:
                     output=self.plan.mask,
                     rotation_units='rad'
                 )
-                self.plan.mask_padded[cxv - cxt:cxv + cxt + mx,
-                                      cyv - cyt:cyv + cyt + my,
-                                      czv - czt:czv + czt + mz] = self.plan.mask
+                self.plan.mask_padded[pad_index] = self.plan.mask
```

Comment:
these lines https://github.com/SBC-Utrecht/pytom-match-pick/blob/1fbbfb6ea8ecbe969ac558fa189e61118890e742/src/pytom_tm/matching.py?plain=1#L205-L218 are not covered by tests, please add a test (even if it wasn't before)

Reply:
that is for non-spherical masks, i'll make sure they are covered, should be a very quick fix

Reply:
this is still not hit

Reply:
Set the wrong value haha
Hope it is fixed now